classes/controller/FrontController.php

```diff
@@ -393,11 +393,11 @@ public function init()
 
         /* Cart already exists */
         if ((int) $this->context->cookie->id_cart) {
-            if (!isset($cart)) {
+            if (!isset($cart) || !Validate::isLoadedObject($cart)) {
```

Comment:
So we re-create the cart object if the previous cart object cannot be loaded

I'm a little worried that then we would load some cart content 🤔  do you know how this usecase can happen?

Reply:
For me it is mainly a security in case the global $cart is upset.